lib/mayaUsd/resources/scripts/mayaUsdAddMayaReference.mel

```diff
@@ -690,6 +735,18 @@ global proc addMayaReferenceToUsdInitUi(string $parent, string $filterType)
     fileOptionsSetup($parent, false, $filterType);
 }
 
+global proc addMayaReferenceToUsdSelectionChanged(string $parent, string $selection)
+{
+    global string $gCurrentFileDialogSelection;
+
+    setParent $parent;
+    $gCurrentFileDialogSelection = $selection;
+
+    // Update the variant preview field.
+    string $groupPrimName = `textFieldGrp -q -text usdMayaReferenceGroupPrimName`;
+    usdMayaRef_changeGroupPrimNameText($groupPrimName);
+}
```

Comment:
New proc called when user selects a file in the dialog. So we can update the variant preview field with the group prim name (if enabled).